src/main/java/core/basesyntax/dao/impl/MessageDetailsDaoImpl.java

```diff
@@ -11,11 +13,34 @@ public MessageDetailsDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public MessageDetails create(MessageDetails entity) {
-        return null;
+
+        Transaction transaction = null;
+        Session session = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.save(entity);
```

Comment:
```suggestion
            session.save(entity);
```
https://mate-academy.github.io/jv-program-fulltime/03_hibernate/checklist/hibernate_02.html

Reply:
done